evm/check-ownership-request.js

```diff
@@ -0,0 +1,194 @@
+'use strict';
+
+require('dotenv').config();
+
+const axios = require('axios');
+const { Command, Option } = require('commander');
+const { ethers } = require('hardhat');
+const { Contract, getDefaultProvider } = ethers;
+const { loadConfig, validateParameters, printError, getContractJSON, printInfo, printWarn, printObj } = require('./utils');
+
+const interchainTokenFactoryABI = getContractJSON('InterchainTokenFactory').abi;
+const interchainTokenABI = getContractJSON('InterchainToken').abi;
+const interchainTokenServiceABI = getContractJSON('InterchainTokenService').abi;
+const erc20ABI = getContractJSON('IERC20Named').abi;
+
+async function processCommand(config, options) {
+    try {
+        const { deployer, address, its, rpc, api } = options;
+        let { source, destination } = options;
+
+        validateParameters({ isValidAddress: { address }, isNonEmptyString: { source, destination } });
```

Comment:
If destination is meant to be an array of strings then you should probably use [this](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/utils.js#L203) function instead, but make sure to move
`destination = JSON.parse(destination);`
above this check and also wrap the `JSON.parse` in a `try/catch` block in case the input cannot be parsed as a JSON